lib/options.go

```diff
@@ -428,6 +430,9 @@ func (o Options) Apply(opts Options) Options {
 	if opts.ConsoleOutput.Valid {
 		o.ConsoleOutput = opts.ConsoleOutput
 	}
+	if opts.Nic.Valid {
+		o.Nic = opts.Nic
```

Comment:
Can you just add a very simple test here, to make sure we're overriding this option correctly: https://github.com/loadimpact/k6/blob/7b6b2a684b7f26bbcf1b8549db89614ac398ad54/lib/options_test.go#L399-L405

Reply:
Thanks for the sample code.

I first wrote the tests to fail and ran the test suite to confirmed failure:

```
opts := Options{}.Apply(Options{Nic: null.StringFrom("eth0")})
assert.Equal(t, "foobar", opts.Nic.String)
```

Then I updated the test and re-ran the suite to confirm success. Committed!